src/usb.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2023 GNOME Foundation Inc.
+ *             2020 Endless OS Foundation LLC
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authors:
+ *       Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
+ *       Ryan Gonzalez <rymg19+github@gmail.com>
+ */
+
+#pragma once
+
+#include <gio/gio.h>
+
+GDBusInterfaceSkeleton * usb_create (GDBusConnection *connection,
+                                     const char      *dbus_name);
+
+void revoke_usb_devices_from_sender (const char *sender);
```

Comment:
Let's namespace this properly, so `usb_revoke_devices_from_sender`

It also doesn't revoke anything, does it? It just cleans up things when a client disconnects.

Reply:
renamed for now.